kupfer/plugin/id2.py

```diff
@@ -0,0 +1,135 @@
+""" ID2 – Guess Identifier (Kupfer Plugin). See https://id2.dev
+"""
+__kupfer_name__    = ("Identifier Resolver")
+__kupfer_sources__ = ()
+__kupfer_actions__ = ( "LookupID", )
+__description__    = ("""Detect and open DOIs, ISBNs, OrcIDs, crypto-addresses, geo-coordinates, and more in your web-browser. Examples:
+
+- Unicode:  \t"U+1F194"
+- UnixTime: \t"1606324253"
+- Location: \t"8FWH4HX8+QR"
+- DOI:    \t\t"10.1109/FDL.2018.8524068"
+- OrcID:    \t"0000-0002-0006-7761"
+- BitCoin:  \t"3FZbgi29cpjq2GjdwV8eyHuJJnkLtktZc5"
+
+For a full list, see https://id2.dev
+
+To disable a certain ID, write "x" into URL field below.
+""")
+__version__ = "2021.0"
+__author__  = "Emanuel Regnath"
+
+
+import urllib.parse
+import re
+
+from kupfer.objects import Action, TextLeaf
+from kupfer import utils, plugin_support
+from kupfer.plugin.id2_support import id2data   # Dict/JSON with all IDs
```

Comment:
I think it would be best to load a static JSON file. This would properly separate the data from the logic of this plugin. Where should I place such a file and how do I get the filepath within my plugin? Is loading static files from a plugin allowed? I think the other two options are suboptimal:

1. copy it into `id2.py`: this would make the file very long, hard to read and mixes data with logic.
2. extra package: since it is just JSON data, creating a python package for it also feels wrong. As Karol pointed out it should then be formatted as proper python code but I would like to keep it as JSON data (because that is what it is).